### ISSUE TITLE
Docker support

### ISSUE THREAD

**Issue by balupton** [scrimba/community#359]
It would be nice if we could specify a `Dockerfile` which can install system dev deps. This would knock off all of these in one swift sweep:

https://github.com/scrimba/community/issues/327
https://github.com/scrimba/community/issues/338
https://github.com/scrimba/community/issues/297
https://github.com/scrimba/community/issues/189
https://github.com/scrimba/community/issues/87

**Comment by hamzamohdzubair** [comment]
Any update?

**Comment by mattjanus** [comment]
Amazon is also a possibility. They're currently offering free services too. Perhaps a user gives you access to their amazon account, you have an automation to standup the stack and ties it into your platform. you could probably use github workflows to take care of the deployment stuff as the user saves.

**Comment by Pistus** [comment]
Good idea @mattjanus - we are looking into if we can use serverless or container orchestration to support backend languages. But it is a big undertaking and we need to get our fundation in place as a small team.
